src/classes/NativeLoader.ts

```diff
@@ -26,6 +26,9 @@ export interface NativeLwk {
   createSigner(mnemonic: string, network: Network): string;
   sign(signerId: string, psetId: string): string;
   wpkhSlip77Descriptor(signerId: string): string;
+  keyoriginXpub(signerId: string, bipId: string): string;
+  createRandomSigner(network: Network): string;
```

Comment:
Why do we need random signer?

Reply:
I understand the reason why we need it. Create random signer can be created without the need of mnemonic.!